compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/LightTreePositioningStrategies.kt

```diff
@@ -409,12 +409,16 @@ object LightTreePositioningStrategies {
             endOffset: Int,
             tree: FlyweightCapableTreeStructure<LighterASTNode>
         ): List<TextRange> {
-            //PSI counterpart simply search for first RPAR descendant, but this one is more correct
-            return tree.findDescendantByType(node, KtNodeTypes.VALUE_ARGUMENT_LIST)?.let { valueArgumentList ->
+            val nodeToStart = when (node.tokenType) {
+                KtNodeTypes.DOT_QUALIFIED_EXPRESSION, KtNodeTypes.SAFE_ACCESS_EXPRESSION ->
```

Comment:
I was curious why `SAFE_ACCESS_EXPRESSION` is here in LT positioning strategy while the counterpart is not there in PSI positioning strategy, and found that `KtSafeQualifiedExpression` is a subtype of `KtQualifiedExpression` (and that's why it's not necessary there).

Then I searched for similar usages:
* https://github.com/JetBrains/kotlin/blob/master/compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/LightTreePositioningStrategies.kt#L457

and found:
https://github.com/JetBrains/kotlin/blob/master/compiler/fir/raw-fir/light-tree2fir/src/org/jetbrains/kotlin/fir/lightTree/converter/ConverterUtil.kt#L43

Not sure it's available right here. Otherwise, we can utilize it somehow. Anyway, it'd be better to use these in a unified, consistent way, instead of these manual lists.

Reply:
OK, it's not available from here. But let me put such a set inside `KtTokens` and use the new set.

Reply:
Cool!

While you're on it, could you update line 460 too? It could be:
```kt
if (node.tokenType not in KtTokens.QUALIFIED_ACCESS) ...
```
?